march_shared_classes/src/march_shared_classes/gait/subgait.py

```diff
@@ -15,7 +15,7 @@ class Subgait(object):
     joint_class = JointTrajectory
 
     def __init__(self, joints, duration, gait_type='walk_like', gait_name='Walk', subgait_name='right_open',
-                 version='First try', description='Just a simple gait'):
+                 version='First try', description='Just a simple gait', *args):
```

Comment:
```suggestion
                 version='First try', description='Just a simple gait', *args):
```
I believe this *args is not necessary (also wasn't in there before). Also remove the one in the call in line 113